src/aasthaJJun20/ArraysUnionIntersection.java

```diff
@@ -0,0 +1,80 @@
+package aasthaJJun20;
+
+public class ArraysUnionIntersection {
+	// This method is written just to display the array elements on console
+	private void displayArray(double[] input) {
+		System.out.print("{ ");
+		for (int index = 0; index < input.length; index++) {
+			System.out.print(input[index] + "   ");
+		}
+		System.out.println("}");
+	}
+
+	// Below method is written to search a particular element in the provided(input)
+	// array
+	private boolean searchInArray(double[] input, double element) {
+		for (int index = 0; index < input.length; index++) {
+			if (input[index] == element)
+				return true;
+		}
+		return false;
+	}
+
+	// Program 1: Find union of two arrays.
+	public double[] getUnionOfArray(double[] arr1, double[] arr2) {
+		double[] unionArray = new double[arr1.length + arr2.length];
+		int size = 0;
+		for (int index = 0; index < arr1.length; index++) {
+			if (searchInArray(unionArray, arr1[index]) == false) {
+				unionArray[size] = arr1[index];
+				size++;
+			}
+		}
+		for (int index = 0; index < arr2.length; index++) {
+			if (searchInArray(unionArray, arr2[index]) == false) {
```

Comment:
if (searchInArray(unionArray, arr1[index],size) == false) {

pass size variable here, why do we keep compare all values.